Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -491,6 +497,11 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
             /// Returns true if any of the objects inside QueryBuffer contain a grabbable
             /// </summary>
             public bool ContainsGrabbable => grabbable != null;
+
+            /// <summary>
+            /// Returns true if any of the objects inside QueryBuffer contain a grabbable that is not a bounds handle
+            /// </summary>
+            public bool NearObjectDetected => ContainsGrabbable && !grabbable.IsBoundsHandles;
```

Comment:
This looks like extra debug stuff also, maybe part of another pr?

Reply:
Oh this is actually used in lines 171

public bool IsNearObject => queryBufferNearObjectRadius.ContainsGrabbable || queryBufferInteractionRadius.NearObjectDetected;

Its basically here to turn off near object interaction if the object is in the grabbable sphere cast range, since that edge case can occur when the near object detection is a sector. It also ignores bound handlers to avoid the bug we had experienced before with flickering handlers.